tests/acceptance/features/cliCommands/trashBin.feature

```diff
@@ -92,3 +92,47 @@ Feature: trashbin
       | resource          | type   |
       | /testfile.txt     | file   |
       | /folder-to-delete | folder |
+
+
+  Scenario: purge expired trash resource of Personal space
+    Given the following configs have been set:
+      | config                                               | value |
+      | STORAGE_USERS_PURGE_TRASH_BIN_PERSONAL_DELETE_BEFORE | 5s    |
```

Comment:
Is `5s` minimum?

Reply:
no

Reply:
then maybe we can use `1s` or `2s` to reduce the scenario wait time